packages/mcp/tsconfig.json

```diff
@@ -5,14 +5,22 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "strictNullChecks": true,
-    "plugins": [{ "transform": "@salesforce/core/lib/messageTransformer", "import": "messageTransformer" }],
     "skipLibCheck": true
   },
   "include": ["./src"],
   "references": [
     {
       "path": "../mcp-provider-api/tsconfig.json"
     },
+    {
+      "path": "../mcp-provider-dx-core/tsconfig.json"
+    },
+    {
+      "path": "../mcp-provider-devops/tsconfig.build.json"
+    },
+    {
+      "path": "../mcp-provider-mobile-web/tsconfig.build.json"
+    },
```

Comment:
more project references.

this is needed if none of the packages are build and you run `yarn build` inside `packages/mcp`, it will build all the references before compiling the mcp source